core/src/epicli/data/common/defaults/configuration/image-registry.yml

```diff
@@ -7,181 +7,230 @@ specification:
+          file_name: operator-1.8.1.tar
+        # postgres
+        - name: bitnami/pgpool:4.1.1-debian-10-r29
+          file_name: pgpool-4.1.1-debian-10-r29.tar
+        - name: "brainsam/pgbouncer:1.12"
+          file_name: pgbouncer-1.12.tar
+      current:
+        - name: "haproxy:2.2.2-alpine"
+          file_name: haproxy-2.2.2-alpine.tar
+        - name: "k8s.gcr.io/kube-apiserver:v1.18.6"
+          file_name: kube-apiserver-v1.18.6.tar
+        - name: "k8s.gcr.io/kube-controller-manager:v1.18.6"
+          file_name: kube-controller-manager-v1.18.6.tar
+        - name: "k8s.gcr.io/kube-proxy:v1.18.6"
+          file_name: kube-proxy-v1.18.6.tar
+        - name: "k8s.gcr.io/kube-scheduler:v1.18.6"
+          file_name: kube-scheduler-v1.18.6.tar
+        - name: "k8s.gcr.io/coredns:1.6.7"
+          file_name: coredns-1.6.7.tar
+        - name: "k8s.gcr.io/etcd:3.4.3-0"
+          file_name: etcd-3.4.3-0.tar
+        - name: "k8s.gcr.io/pause:3.2"
+          file_name: pause-3.2.tar
+        # flannel
+        - name: "quay.io/coreos/flannel:v0.12.0-amd64"
+          file_name: flannel-v0.12.0-amd64.tar
+        - name: "quay.io/coreos/flannel:v0.12.0"
+          file_name: flannel-v0.12.0.tar
+        # canal & calico
+        - name: "calico/cni:v3.15.0"
+          file_name: cni-v3.15.0.tar
+        - name: "calico/kube-controllers:v3.15.0"
+          file_name: kube-controllers-v3.15.0.tar
+        - name: "calico/node:v3.15.0"
+          file_name: node-v3.15.0.tar
+        - name: "calico/pod2daemon-flexvol:v3.15.0"
+          file_name: pod2daemon-flexvol-v3.15.0.tar
+      legacy:
+      # v1.14.6 - Epiphany >= 0.4.0 && < 0.6.0
+        - name: "k8s.gcr.io/kube-apiserver:v1.14.6"
+          file_name: kube-apiserver-v1.14.6.tar
+        - name: "k8s.gcr.io/kube-controller-manager:v1.14.6"
+          file_name: kube-controller-manager-v1.14.6.tar
+        - name: "k8s.gcr.io/kube-scheduler:v1.14.6"
+          file_name: kube-scheduler-v1.14.6.tar
+        - name: "k8s.gcr.io/kube-proxy:v1.14.6"
+          file_name: kube-proxy-v1.14.6.tar
+        - name: "k8s.gcr.io/etcd:3.3.10"
+          file_name: etcd-3.3.10.tar
+        - name: "k8s.gcr.io/coredns:1.3.1"
+          file_name: coredns-1.3.1.tar
+        - name: "coredns/coredns:1.5.0"
+          file_name: coredns-1.5.0.tar
+        - name: "k8s.gcr.io/pause:3.1"
+          file_name: pause-3.1.tar
+        - name: "quay.io/coreos/flannel:v0.11.0-amd64"
+          file_name: flannel-v0.11.0-amd64.tar
+        - name: "quay.io/coreos/flannel:v0.11.0"
+          file_name: flannel-v0.11.0.tar
+        - name: "calico/cni:v3.8.1"
+          file_name: cni-v3.8.1.tar
+        - name: "calico/kube-controllers:v3.8.1"
+          file_name: kube-controllers-v3.8.1.tar
+        - name: "calico/node:v3.8.1"
+          file_name: node-v3.8.1.tar
+        - name: "calico/pod2daemon-flexvol:v3.8.1"
+          file_name: pod2daemon-flexvol-v3.8.1.tar
```

Comment:
I think we should remove those already.

Reply:
I dont think we can remove these at this point because of upgrading process. For aarch64 I did not include that stuff as we dont have it anything to upgrade jet;)

Reply:
We no longer support `< 0.6.0` so I would remove it or at least comment it out and if someone really needs it he/she could uncomment.

Reply:
Yeah I saw that is deprecated so we can concider it. I have added a seperate issue (#2179) as I feel this is not something that should be in this pull. Will require different testing with upgrades etc which this story does not.